agent_config.go

```diff
@@ -100,6 +103,11 @@ type AgentConfig struct {
 	// keepAlive behavior should be modified with KeepaliveInterval and ConnectionTimeout
 	taskLoopInterval time.Duration
 
+	// taskLoopInterval controls how often our internal task loop runs when in the connecting phase,
+	// this would normally be more frequent thank the normal "taskLoopInterval", to allow
```

Comment:
The code snippet I posted at https://github.com/pion/ice/pull/201#discussion_r446602110 make it possible to remove `taskLoopInterval` and users no longer have to consider `be more frequent thank the normal "taskLoopInterval"`.

Reply:
It may be a different PR, and I'll do it if you are Ok.